src/components/Modals/EditProfileModal.vue

```diff
@@ -0,0 +1,190 @@
+<template>
+  <our-modal @close-modal="closeModal()" :open="true" size="large">
+    <template v-slot:header>
+      <p class="text-4xl">Edit Account</p>
+    </template>
+    <template v-slot:content>
+      <ValidationObserver v-slot="{ invalid, handleSubmit }">
+        <form @submit.prevent="handleSubmit(submit)">
+          <div class="flex mt-6 social-links">
+            <div class="w-1/3 flex">
+              <label for="profile-image" class="w-40 h-32 block cursor-pointer">
+                <input type="file" name="profile image" id="profile-image" class="hidden" accept="image/*" @change="onFileUpload($event);">
+                <div class="w-full h-full profile-image flex items-center justify-center" v-if="!uploadedImageSrc">
+                  <span class="text-white text-sm font-circular">Upload picture</span>
+                </div>
+                <!-- thinking of using canvas here so I can make modifications to the image after upload -->
+                <div v-bind:style="{ 'background-image': 'url('+ uploadedImageSrc +')'  }" class="w-full h-full bg-cover bg-no-repeat"></div>
+              </label>
+            </div>
+            <div class="w-2/3">
```

Comment:
How come there's nothing in this tag?

Reply:
I removed the input search that was there and I am using that div as a structural element.

Reply:
Alright 👍